### PR TITLE
Fix setting of base_output_dir

### PR DESCRIPTION
**Pull request by aarontp**
... Also fix result.close() in artifacts.py

### CODE REVIEW

**Comment by aarontp** [comment]
FYI, I also added the string conversions so tests would pass.

File: turbinia/output_manager.py [hunk 1]
```diff
@@ -172,12 +172,17 @@ class OutputWriter(object):
 
   NAME = 'base_output_writer'
 
-  def __init__(self, unique_dir=None, local_output_dir=None):
```
Comment (Onager) [inline turbinia/output_manager.py]: I think it's worth adding a docstring here, to mention the auto-setting of some of these params

Comment (aarontp) [reply turbinia/output_manager.py]: Done.

**Comment by aarontp** [comment]
PTAL

**Pull request merged**